README.rst

```diff
@@ -13,37 +13,42 @@ Micro-frontend for the single-page payment/checkout process. This application su
 Getting Started
 ---------------
 
-**Prerequisite**
+This MFE is bundled with `Devstack <https://github.com/edx/devstack>`_, see the `Getting Started <https://github.com/edx/devstack#getting-started>`_ section for setup instructions.
 
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
+Once set up, this frontend and all its prerequisites can be started with:
 
-**Installation and Startup**
+.. code-block::
 
-1. Clone this repo:
+  cd devstack
+  make dev.up.frontend-app-payment
 
-  ``git clone https://github.com/edx/frontend-app-payment.git``
+The frontend runs at `http://localhost:1998 <http://localhost:1998>`_, though it is rare to interact with it directly.  Rather, log into the LMS at http://localhost:18000/login and enroll in a course, choosing "upgrade to verified" to be taken to the checkout page.
 
-2. Install npm dependencies:
-
-  ``cd frontend-app-payment && npm ci``
+Development
+-----------
 
-  Note: ``npm ci`` is recommended over ``npm install`` to match the way CI and production builds work and avoid unintentional changes to ``package_lock.json`` when doing other work.
+To work on this frontend: with the prerequisites running as above, bring down the ``frontend-app-payment`` container, then start the development server *outside* Docker as follows:
 
-3. Start the dev server:
+.. code-block::
 
-  ``npm start``
+  cd devstack
+  make dev.down.frontend-app-payment
+  cd ../frontend-app-payment
+  npm ci
+  npm start
 
-The dev server is running at `http://localhost:1998 <http://localhost:1998>`_.
+Note: ``npm ci`` is recommended over ``npm install`` to match the way CI and production builds work and avoid unintentional changes to ``package_lock.json`` when doing other work.
 
-By default it will show an empty basket view.
+The dev server also runs at `http://localhost:1998 <http://localhost:1998>`_, but again, it is rare to interact with it directly.  By default it will show an empty basket view.
 
-**Enable in devstack**
+Enable in ecommerce
+-------------------
 
-To have your devstack ecommerce use this dev server:
+Devstack ecommerce has this frontend enabled by default, to enable it in another installation:
```

Comment:
Question: what does it mean to `enable it in another installation`?

Reply:
A production installation for example.  This used to be how to configure it in devstack before it was bundled by default and I think it's important information; maybe it should be restructured into an "environment variables" section.

Reply:
If you have the capacity to do so, that'd be awesome. Otherwise I might say `another installation (for example, a production installation)`